3303Elevator/src/app/FloorSubsystem/FloorSubsystem.java

```diff
@@ -212,29 +286,103 @@ private static void runCommandLineUI(Scanner sc) { //, Scheduler scheduler has b
 	    		 
 	    		 //TODO : THIS GOTTA BE TAILORED TO FLOOR SUBSYSTEM MAKING INPUT REQUESTS
 	    	 } else if (next.equals("n")) {
-	    		 System.out.println("Enter your command with the following format <CurrentFloor> <DestinationFloor>\n"+"Example: 2 6 for a request to go from floor 2 to 6");
+	    		 System.out.println("Enter your command with the following format <Time in milliseconds delay or Timestamp 'hh:mm:ss'> <CurrentFloor> <Direction> <DestinationFloor>\n"+"Example: 1000,5,Down,2 for a request to go from floor 5 to 2 after 1000 milliseconds");
+	    		 System.out.println("Example: 22:51:00,5,Down,2 for a request to go from floor 5 to 2 at 22:51");
```

Comment:
Add an example string with milliseconds
Also build the strings and use one printout instead of multiple printouts. Avoid other threads printing in between them

Reply:
the one for milliseconds is in line 289